src/MessagePack.UnityClient/Assets/Scripts/MessagePack/Resolvers/StandardResolver.cs

```diff
@@ -289,10 +289,6 @@ internal static class StandardResolverHelper
             MessagePack.Unity.UnityResolver.Instance,
 #endif
 
-#if !ENABLE_IL2CPP && !NET_STANDARD_2_0
-            DynamicEnumResolver.Instance, // Try Enum
```

Comment:
@neuecc Is there any risk that this will break scenarios that work today, for which we may be lacking test coverage?

Reply:
I think it's okay.